Cython/Compiler/Options.py

```diff
@@ -373,6 +374,7 @@ def normalise_encoding_name(option_name, encoding):
     'trashcan' : ('cclass',),
     'total_ordering': ('cclass', ),
     'cpp_locals': ('module', 'function', 'cclass'),  # I don't think they make sense in a with_statement
+    'hpy': ('module', 'function'),
```

Comment:
Is the idea to let HPy be set on a per-function basis (and if so, how do classes and member functions fit into this)?

It looks from the C code like HPy is either on or off?

Reply:
@fangerer, IIRC, this is just a work-around for the incomplete syntax coverage, right?

Reply:
That's basically a left-over. @mattip started with that approach such that he could select for which functions the HPy code is generated. I think we should just remove that (I'll do that soon).

Reply:
I would propose removing this as a last step in the integration of HPy into Cython. It provides a migration path so that we can merge smaller PRs.